src/api/entities/SecurityToken/Checkpoints/__tests__/index.ts

```diff
@@ -88,15 +88,17 @@ describe('Checkpoints class', () => {
 
       const requestPaginatedStub = sinon.stub(utilsInternalModule, 'requestPaginated');
 
+      const fakeTicker = 'FAKETICKER';
+
       const totalSupply = [
         {
-          ticker: 'FAKETICKER',
+          ticker: fakeTicker,
           checkpointId: new BigNumber(1),
           balance: new BigNumber(100),
           moment: new Date('10/10/2020'),
         },
         {
-          ticker: 'OTHERTICKER',
+          ticker: fakeTicker,
```

Comment:
Why is it still a property in each entry? You can use the variable directly later on instead of getting it from the entries in the array

Reply:
Same for rawTicker, you can just define it once and then use it where needed